src/main/kotlin/me/modmuss50/mpp/platforms/modrinth/Modrinth.kt

```diff
@@ -154,10 +154,17 @@ interface ModrinthDependency : PlatformDependency {
 
     @get:Input
     @get:Optional
-    val version: Property<String>
+    val versionId: Property<String>
+
+    @get:Input
+    @get:Optional
+    val versionSlug: Property<String>
 
     @Deprecated("For removal", ReplaceWith("id"))
     val projectId: Property<String> get() = id
+
+    @Deprecated("For removal", ReplaceWith("versionId"))
+    val version: Property<String> get() = versionId
```

Comment:
I wonder if just using the existing `version` for both would be fine. You can use the new request to validate that the specided version id exists, if not try to find it via the version name.